packages/editor/src/components/post-card-panel/index.js

```diff
@@ -106,3 +129,59 @@ export default function PostCardPanel( { className, actions } ) {
 		</PanelBody>
 	);
 }
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+// This component renders the wordcount and reading time for the post.
+function PostContentInfo( { postContent } ) {
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const wordsCounted = postContent
+		? wordCount( postContent, wordCountType )
+		: 0;
+	if ( ! wordsCounted ) {
+		return null;
+	}
+	const readingTime = Math.round( wordsCounted / AVERAGE_READING_RATE );
+	let wordsCountText;
+	if ( ! wordsCounted.toLocaleString() ) {
+		wordsCountText = __( 'Unknown' );
+	} else {
+		wordsCountText =
+			wordsCounted === 1
+				? __( '1 word' )
+				: sprintf(
+						// translators: %s: the number of words in the post.
+						_n( '%s word', '%s words', wordsCounted ),
+						wordsCounted.toLocaleString()
+				  );
+	}
+	const readingTimeText =
+		readingTime <= 1
+			? __( '1 minute read time' )
+			: sprintf(
+					// translators: %s: the number of words in the post.
+					_n(
+						'%s minute read time',
+						'%s minutes read time',
+						readingTime
+					),
+					readingTime.toLocaleString()
+			  );
+	return (
+		<Text>
+			{ sprintf(
+				/* translators: 1: How many words a post has (e.g. 30 words). 2: the number of minutes of read time (e.g. 2 minutes read time) */
+				/* translators: %1s: is the number of minutes. */
+				'%1$s, %2$s.',
```

Comment:
Usually such strings just containing placeholder are not really useful for translators.

Could we maybe move it around a little bit?

`%1$s. %2$s read time.`

Reply:
> Usually such strings just containing placeholder are not really useful for translators.

This was initially my suggestion, based on the idea that translators ought to decide which, if any, punctuation should best separate the two fragments. Of course, in a non-numerical setting this could all be a single string, e.g. `__( '%1$s words, %2$s reading time.' )`, but since we need to take care of number agreement foremost this seemed like a reasonable concatenation technique.

Reply:
With something like `%1$s. %2$s read time.` we can still deal with the numbers in the individual strings, e.g. `%1$s` is `_n( '%s word', '%s words'` and `%2$s` here is `_n( '%s minute', '%s minutes' )`

Reply:
I updated per @swissspidy feedback.